config/core/resources/channel.yaml

```diff
@@ -36,293 +36,189 @@ spec:
     shortNames:
       - pschan
   scope: Namespaced
-  subresources:
-    status: {}
   preserveUnknownFields: false
   conversion:
     strategy: Webhook
-    webhookClientConfig:
+    # conversionReviewVersions indicates what ConversionReview versions are understood/preferred by the webhook.
+    # The first version in the list understood by the API server is sent to the webhook.
+    # The webhook must respond with a ConversionReview object in the same version it received.
```

Comment:
I think you are missing something. The e2e failed with applying this yaml.  https://github.com/bharattkukreja/knative-gcp/blob/dee416986d74f3cf671ab2f7e25264b54c190c47/config/core/resources/cloudpubsubsource.yaml#L42